modules/cql/test/blaze/elm/concept_spec.clj

```diff
@@ -0,0 +1,15 @@
+(ns blaze.elm.concept-spec
+    (:require
+      [blaze.elm.concept :as concept]
+      [clojure.spec.alpha :as s])
+    (:import
+      [blaze.elm.concept Concept]))
+
+
+(defn concept? [x]
+  (instance? Concept x))
+
+
+(s/fdef concept/to-concept
+  :args (s/coll-of (s/cat :system string? :version (s/nilable string?) :code string?))
```

Comment:
The :args spec almost always [starts with `s/cat`](https://clojure.org/guides/spec#_specing_functions) so it will be `(s/cat :codes ...)`. For the "codes" spec, I would take `(s/coll-of code-spec/code?)` in order to test for a compiled code. For that to work, you need to require `[blaze.elm.code-spec :as code-spec]`.